includes/Plugins.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace NewfoldLabs\WP\Module\AI;
+
+use NewfoldLabs\WP\Module\Installer\Services\PluginInstaller;
```

Comment:
Shall we add the installer module as a dependency in composer?

Reply:
I had thought of adding it as composer dependency but then it felt like adding one more point of failure for the module. Incase the `wp-module-installer` gets updated then all the modules which have it as a dependency will need to update the versions else they might break. Lets discuss this offline.

Reply:
updated composer dependency